system/ee/language/english/fieldtypes_lang.php

```diff
@@ -75,19 +75,19 @@
 
     'rel_ft_display_entry_id_desc' => 'When enabled, entry IDs will be displayed together with entry title inside the field.',
 
-    'rel_ft_max' => 'Maximum selection',
+    'rel_ft_max' => 'Maximum relationship',
 
     'rel_ft_max_desc' => 'Sets the maximum number of entries this field can be used to relate.',
 
     'rel_ft_max_error' => 'You can select no more than %d entries.',
 
-    'rel_ft_min' => 'Minimum selection',
+    'rel_ft_min' => 'Minimum relationship',
 
     'rel_ft_min_desc' => 'Sets the minimum number of entries this field can be used to relate',
 
     'rel_ft_min_error' => 'You need to select at least %d entries.',
 
-    'rel_ft_min_settings_error' => 'The minimum selection cannot be set to number less than maximum entries to be shown',
+    'rel_ft_min_settings_error' => 'The minimum relationship cannot be set to number less than maximum entries to be shown',
```

Comment:
Wondering if we'd better use plural here? E.g. `Maximum relationships`

Reply:
I actually don't think we should throw this error. While I don't think it's ideal, I can find more entries than what's initially displayed by using the keyword search. I don't see a reason to require the minimum related entries to under what is initially disaplyed.